stock_quantity_history_location/i18n/stock_quantity_history_location.pot

```diff
@@ -0,0 +1,43 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* stock_quantity_history_location
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 15.0\n"
```

Comment:
16.0.

Please regenerate the template file

Reply:
@luisg123v FYI the pot file is regenerated at merge and usually is not required to be changed on PR's

Reply:
@rousseldenis thanks for pointing it out. I was aware `.po` files were regenerated, but I didn't know about `.pot` files also were, I though `.po` were regenerated from the provided `.pot`.

Then this LGTM.

Reply:
The .pot is regenerated at merge (of any kind) as if some code has been modified, pot file should be synced with current code. po files are only modified by weblate modifications (several timea per day).